.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,39 +1,17 @@
-* *Bugs: if the issue is intermittent and not easily reproducible, describe the situation/scenarios/configurations where it was likely triggered*
-* *If there is an issue ID, link to it: [PP-XXXX](https://pbspro.atlassian.net/browse/PP-XXXX)*
+#### Bug or Feature Description
+* *Describe the problem, ideally from the customer's viewpoint*
 
-#### Affected Platform(s)
-* *Platform (for example CentOS 7.2)*
+#### Description of Your Change
+* *Say how you fixed the problem.  Please describe your code changes in detail for reviewers*
 
-#### Cause / Analysis / Design
+#### Link to Design
 * *If there is an external design, link to it **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)***
-* *If there is a discussion on the Forum, link to it **[pbspro community forum](http://community.pbspro.org/)***
-* *Bugs: briefly describe root cause/analysis of the problem*
 
-#### Solution Description
-* *Please describe your code changes in detail for reviewers*
-
-#### Testing logs/output
+#### Testing Logs or Output
 * *Please attach your test log output from running the test you added (or from existing tests that cover your changes)*
 
-#### Checklist:
-<!--- Use the preview button to see the checkboxes/links properly. -->
-<!--- Go over the following points, and put an `x` (without spaces around it) in the boxes that apply. -->
-<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
-- [ ] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
-- [ ] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
-- [ ] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
-- [ ] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
-   - [ ] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
-- [ ] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
-   - [ ] I have added  **manual test(s) to this pull request and explained why PTL is not appropriate** for this case.
-- [ ] All new and existing automated tests have passed. (See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**).
-- [ ] I have attached **test logs to this pull request** as evidence of testing/verification.
-
-
-__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__
+#### Optional Considerations
```

Comment:
Actually, many of the items in the "optional consideration" table on the link aren't really optional. We require people to follow our coding standards, we require the code header to contain license text, etc. So, maybe we should call this "Guidelines for first time contributors", or just "Pull Request Guidelines" ? And I think there's some stuff redundant between the template and that confluence page, maybe we should remove that. We should also remove commit signing from that page now that it's not needed.

Reply:
I agree some are actually not optional, and we should remove all mention of commit signing.